Provis.Core/Services/TaskService.cs

```diff
@@ -55,5 +65,50 @@ public async Task<List<TaskDTO>> GetUserTasksAsync(string userId, int workspaceI
             var mapTask = _mapper.Map<List<TaskDTO>>(userTasks);
             return mapTask;
         }
+
+        public async Task CreateTaskAsync(TaskCreateDTO taskCreateDTO, string userId)
```

Comment:
Create tech dept ticket for make transaction logic and integrate it here

Reply:
https://app.clickup.com/t/1tf8jxe